Lib/_colorize.py

```diff
@@ -138,26 +209,25 @@ def can_colorize(*, file: IO[str] | IO[bytes] | None = None) -> bool:
         return hasattr(file, "isatty") and file.isatty()
 
 
-def set_theme(t: dict[ColorTag, str] | None = None) -> None:
-    global theme
+default_theme = Theme()
+theme_no_color = default_theme.no_colors()
+
+
+def get_theme(
+    *,
+    tty_file: IO[str] | IO[bytes] | None = None,
+    force_color: bool = False,
+    force_no_color: bool = False,
+) -> Theme:
+    if force_color or (not force_no_color and can_colorize(file=tty_file)):
```

Comment:
Shall we have `get_colors`, `can_colorize` and `get_theme` all use the same parameter name: either all `tty_file` or all `file`?

It's the same thing, used for the same purpose.